cropper/src/main/java/com/canhub/cropper/BitmapCroppingWorkerJob.kt

```diff
@@ -157,7 +157,7 @@ class BitmapCroppingWorkerJob internal constructor(
                         }
                     }
                     val resizedBitmap =
-                        BitmapUtils.resizeBitmap(bitmapSampled.bitmap, reqWidth, reqHeight, options)
+                        BitmapUtils.resizeBitmap(bitmapSampled.bitmap!!, reqWidth, reqHeight, options)

```

Comment:
why the usage of `!!`  ? sadly this can throw exception. There is another behaviour that we can proceed when this is null?

Reply:
Actually this will not be null... But due to strong type system... Compiler enforces me to handle null checks..